ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -297,6 +310,12 @@ echo "$(date) - waiting for nodes to be ready..."
 oc wait nodes --all --for=condition=Ready=true --timeout=10m
 echo "$(date) - all nodes are ready"
 
+# wait for all clusteroperators to reach progressing=false to ensure that we achieved the configuration specified at installation
+# time before we run our e2e tests.
+echo "$(date) - waiting for clusteroperators to finish progressing..."
+oc wait clusteroperators --all --for=condition=Progressing=false --timeout=10m
+echo "$(date) - all clusteroperators are done progressing."
```

Comment:
Moved to after Nodes checks per @deads2k  request

Reply:
> Moved to after Nodes checks per @deads2k request

Thanks. some operators have work to do *after* nodes are ready, so we want to wait for them to finish.  This looks correct to me.